src/main/java/com/hydratereminder/chat/ChatMessageSender.java

```diff
@@ -49,7 +49,9 @@ public void sendHydrateEmojiChatGameMessage(String message) {
     public void sendHydrateReminderChatMessage(String message) {
         final ChatMessageType chatMessageType = chatMessageTypeProvider.getChatNotificationMessageType();
         sendHydrateEmojiChatMessage(chatMessageType, message);
-        log.debug(String.format("Successfully sent chat notification of type: %s", chatMessageType.toString()));
+        if(log.isDebugEnabled()){
```

Comment:
Can you add some spaces to match the style of the other if statements in the code? `if (...) {`

Reply:
for sure!